tests/test_mdb.py

```diff
@@ -1,5 +1,23 @@
-import os, sys
-sys.path.insert(0,os.path.abspath('./src'))
+import sys, os
+# Execute only if in Python 3.x interpreter
+if sys.version_info[0] > 2:
+    # Check which folder the test are running from and then
+    # add the `src` folder to PATH for testing. Also change
+    # the work directory to `temp` to run Abaqus command in it.
+    folder = os.path.basename(os.getcwd())
+    if folder == 'temp':
+        sys.path.insert(0,os.path.abspath('../../src'))
+    elif folder == 'tests':
+        sys.path.insert(0,os.path.abspath('../src'))
+        os.chdir('temp')
+    else:
+        sys.path.insert(0,os.path.abspath('./src'))
+        os.chdir('tests/temp')
+    
+    #This check is needed to run from `pytest` framework
+    if sys.argv[0] != __file__:
+        sys.argv.insert(0,__file__)
```

Comment:
This is quite complicated, how about use the `pytest_configure` function in the `conftest.py` file (https://stackoverflow.com/a/53690092)? 
```python
def pytest_configure(config):
    os.system('python setup.py install')
```

Reply:
Do you mean, include that block of code in `conftest.py` inside `pytest_configure`?

Reply:
But, if we don't need to submit the tests to Abaqus, we won't need this block of code either

Reply:
If we do not install the `abqpy` package locally, the `pytest` will fail

Reply:
In the GitHub workflows, the `abqpy` will always be installed automatically, in case we have to run `pytest` locally and sometimes we forgot to install it, we might need this.

Reply:
> In the GitHub workflows, the `abqpy` will always be installed automatically, in case we have to run `pytest` locally and sometimes we forgot to install it, we might need this.
Ok, but we won't need the `os.chdir('temp')` anymore, right?

Reply:
yes

Reply:
Done. Removed the modifications in [642d0e2](https://github.com/haiiliin/abqpy/pull/1104/commits/642d0e26c38e8eca004f4af6bba3ac8628436d10). Now the tests doesn't submit to Abaqus.